cg/apps/cgstats/crud/create.py

```diff
@@ -198,39 +198,39 @@ def _calculate_read_counts(demux_sample: DragenDemuxSample) -> int:
 def create_projects(manager: StatsAPI, project_names: Iterable[str]) -> Dict[str, int]:
     project_name_to_id: Dict[str, int] = {}
     for project_name in project_names:
-        project_id: Optional[int] = manager.find_handler.get_project_id(project_name=project_name)
-        if not project_id:
-            project_object: Project = create_project(manager=manager, project_name=project_name)
-            project_id: int = project_object.project_id
-        else:
-            LOG.info("Project %s already exists", project_name)
-        project_name_to_id[project_name] = project_id
+        project: Optional[Project] = get_or_create_project(
+            manager=manager, project_name=project_name
+        )
+        project_name_to_id[project_name] = project.project_id
     return project_name_to_id
 
 
 def _create_samples(
     manager: StatsAPI, sample: NovaSeqSample, project_name_to_id: Dict[str, int]
-) -> Union[int, None]:
-    """handles sample objects creation for the table `Sample` in cgstats"""
+) -> Optional[int]:
+    """Create a new Sample object in the cgstats database if it doesn't already exist."""
+
+    if sample.project == "indexcheck":
+        LOG.debug("Skip adding indexcheck sample to database")
```

Comment:
Maybe this should be a `LOG.warning`or `LOG.info`

Reply:
I'm keeping it as a debug since that was how it was before, just refactoring it.